service/k8sclient/k8sclient_test.go

```diff
@@ -311,17 +372,6 @@ func TestK8sClient(t *testing.T) {
 			assert.Equal(t, err.Error(), fmt.Sprintf(clusterWithSameNameExistsErrTemplate, name))
 		})
 
-		assertListPXCCluster(ctx, t, client, name, func(cluster *PXCCluster) bool {
-			return cluster != nil && cluster.State == ClusterStateReady
-		})
-
-		t.Run("All pods are ready", func(t *testing.T) {
-			cluster, err := getPXCCluster(ctx, client, name)
-			require.NoError(t, err)
-			assert.Equal(t, int32(2), cluster.DetailedState.CountReadyPods())
-			assert.Equal(t, int32(2), cluster.DetailedState.CountAllPods())
-		})
-
```

Comment:
I'm surprised why don't we wait for all pods to be ready?

Reply:
We wait them above